examples/tensorflow/classification/main.py

```diff
@@ -322,6 +322,11 @@ def export(config):
 def main(argv):
     parser = get_argument_parser()
     config = get_config_from_argv(argv, parser)
+
+    if 'cifar' in config.dataset:
+        assert config.dataset[5:].isdigit()
+        config.num_classes = int(config.dataset[5:])
+
```

Comment:
Or we can check it explicitly
```suggestion
    if  config.dataset == 'cifar100':
        config.num_classes = 100
    elif config.dataset == 'cifar10':
        config.num_classes = 10
```

Reply:
I think you should change the default value of the num_classes regarding to dataset here https://github.com/openvinotoolkit/nncf/blob/0938491f35585f8b58dfcfb3175341b5d88eec01/examples/tensorflow/classification/main.py#L150 I suggest doing it in the separate function `def get_num_classes()` in the main.py